usagerep/usagerep.go

```diff
@@ -163,7 +164,7 @@ func (u *UsageReporter) ReportUsage(clusterGUID, clusterType, calicoVersion stri
 	}
 }
 
-func (u *UsageReporter) calculateURL(clusterGUID, clusterType, calicoVersion string, stats calc.StatsUpdate) string {
+func (u *UsageReporter) calculateURL(clusterGUID, clusterType, calicoVersion, alpEnabled string, stats calc.StatsUpdate) string {
```

Comment:
Nit: I'd probably pass the boolean down to this method and then do the `fmt.Sprint()` here.  Reason: minimises the amount of code where you have a wooly `string` instead of a strongly-typed `bool`.